language-server/modules/langserver-core/src/main/java/org/ballerinalang/langserver/extensions/ballerina/syntaxhighlighter/SemanticHighlightKeys.java

```diff
@@ -0,0 +1,33 @@
+/*
+ *  Copyright (c) 2019, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```

Comment:
```suggestion
 *  Copyright (c) 2020, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```
Please correct in all the places

Reply:
I doubt it as these changes were committed in 2019. @praneesha Could you please elaborate you concern? :)

Reply:
We need to change the year in the copyright license of the file to 2020 only if we are adding them in 2020. 

Therefore, we do not need to change it to 2020 for the files that were already added before this year.